apps/vscode/src/host/index.ts

```diff
@@ -112,13 +118,20 @@ function defaultExtensionHost(): ExtensionHost {
       return languages.filter(language => knitr || !visualMode || (language !== "python"));
     },
     cellExecutorForLanguage,
-    // in the default extension host, both of these are just a noop:
-    registerStatementRangeProvider: (engine: MarkdownEngine): vscode.Disposable => {
-      return new vscode.Disposable(() => { });
-    },
-    registerHelpTopicProvider: (engine: MarkdownEngine): vscode.Disposable => {
-      return new vscode.Disposable(() => { });
-    },
-    createPreviewPanel,
+    // In contrast to the Positron-specific `hooksExtensionHost`, here in the default host we
+    // do not have statement range or help topic functionality
+    registerStatementRangeProvider: doNothing,
+    registerHelpTopicProvider: doNothing,
+    createPreviewPanel: (
+      viewType: string,
+      title: string,
+      preserveFocus?: boolean,
+      options?: WebviewPanelOptions & WebviewOptions
+    ): WebviewPanel => {
+      return window.createWebviewPanel(viewType, title, { viewColumn: ViewColumn.Beside, preserveFocus, }, options);
+    }
```

Comment:
This was previously imported from `preview.ts` where it was defined by itself. I found that hard to navigate and also the implementation of this function in the other ExtensionHost in hooks.ts defines this inline.